### ISSUE TITLE
Unicode issue in EE item description

### ISSUE THREAD

**Issue by lynxlynxlynx** [gemrb/gemrb#1924]
Now that inventory works, I noticed we don't render unicode chars, but each byte separately. Take for example the necklace of power or these gauntlets:
https://forums.beamdog.com/uploads/editor/1m/i9eqxahips3l.png
They have a unicode dash when listing item abilities, which we misrender.

**Comment by bradallred** [comment]
We take the 16bit chars and simply split them into an 8bit cycle and 8bit frame for BAM fonts. What do the EEs use for their fonts? IF its TTF, then this might be a great opportunity to do #1048 and/or #1895 which may fix this.

**Comment by lynxlynxlynx** [comment]
We currently use the bam fonts, since they're still there and fonts.2da was untouched. EEs ship a couple of fonts to cater to different i18n needs, so at least NORMAL will have to be changed.

**Comment by MarcelHB** [comment]
There is something like NORMAL.BAM, NORMAL.BMP and NORMAL.TTF.

**Comment by lynxlynxlynx** [comment]
The TTF will be enough for many of them.

This is the closest matching FNT description I can find: http://fileformats.archiveteam.org/wiki/FNT_(Windows_Font)
But already normal.fnt does start with a magic byte, not null.

**Comment by bradallred** [comment]
if we using BAM, then I'm unsure of what we are doing wrong since you only have 16bits to work with. What is the cycle/frame of that dash? could just be they use something that isn't actually unicode.

**Comment by bradallred** [comment]
so you are speculating that those resources are just cruft they never cleaned out?

**Comment by lynxlynxlynx** [comment]
Yes, exactly. Arbitrary resolutions, new ports to handhelds, new languages to support and most importantly, configurable font size, are all much harder with bitmap fonts.

**Comment by lynxlynxlynx** [comment]
We actually do give ttf priority, the file just isn't found, since it is biffed and we don't expect that. Just need to find the magic number. :)

**Comment by lynxlynxlynx** [comment]
Looking at the files, the bam has the usual 255 chars, the bmp extra ones (NI also labels them that way) — interestingly in various sizes. The fnt is indeed non-standard and not well understood, with much more data than presented in NI. 
I found this Japanese modder's tool and notes:
https://forums-beamdog-com.translate.goog/discussion/26576/bgee-font-tool-bgfnt-exe?_x_tr_sl=es&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=wapp
Once SHS is back up this was also suggested reading in the same thread (not present in the Wayback Machine): http://www.shsforums.net/topic/56710-bigger-fonts-for-baldurs-gate/page-2

**Comment by lynxlynxlynx** [comment]
I'll close this an open a new issue, since we now do use the TTF font, but still render an unexpected character.
